doc/preview.yml

```diff
@@ -0,0 +1,57 @@
+site:
+  title: Neo4j Graph Data Science Client
+  url: https://neo4j.com/docs
+  start_page: graph-data-science-client:ROOT:index.adoc
+
+content:
+  sources:
+  - url: ../
+    start_path: doc
+    branches: 
+    - HEAD
+    include: doc/
+    exclude:
+    - '!**/_includes/*'
+    - '!**/readme.adoc'
+    - '!**/README.adoc'
+ui:
+  bundle:
+    url: https://s3-eu-west-1.amazonaws.com/static-content.neo4j.com/build/ui-bundle-latest.zip
+    snapshot: true
+  output_dir: /assets
+
+runtime:
+  fetch: true
+
+urls:
+  html_extension_style: indexify
+
```

Comment:
@recrwplay Do we need to add the antora-modify-sitemaps extension here?

```
antora:
  extensions:
  - require: "@neo4j-antora/antora-modify-sitemaps"
    sitemap_version: '1.2.0'
    sitemap_loc_version: 'current'
    move_sitemaps_to_components: true
```

Reply:
(Just realised that this is the preview, so maybe we don't)

Reply:
In order to publish the sitemap, yes. It can be added now or later.

Reply:
ok, done! :)